en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).

```

Comment:
I will add also **visual** information - Visual-Inertial Odometry or Motion Capture.

Reply:
Isn't this ultimately just position information - albeit obtained from an inertial source?
If so, would the following text be better?:

> * This mode requires position information - e.g. GPS or intertial (optical flow/visual-intertial odometry, etc.)

Reply:
Pint @TSC21

Reply:
Not just position. It can include orientation. Visual or Mocap are not inertial sources (inertial sources integrate IMU data). So it will be **pose**  instead of position:
> **Note** 
>  * This mode requires position or pose information -  e.g. GPS, optical flow, visual-inertial odometry, mocap, etc.